aten/src/ATen/native/LinearAlgebraUtils.h

```diff
@@ -540,8 +540,8 @@ static inline void checkNotComplexTolerance(const Tensor& tol, const c10::string
   This rule is compatible with NumPy, see https://github.com/numpy/numpy/blob/v1.20.0/numpy/linalg/linalg.py#L384-L389
 */
 static inline bool linalg_solve_is_vector_rhs(const Tensor& input, const Tensor& other) {
-  auto expected_batched_rhs_shape = IntArrayRef(input.sizes().data(), input.dim() - 1); // input.shape[:-1]
-  bool vector_case = other.dim() == 1 || (input.dim() - 1 == other.dim() && other.sizes().equals(expected_batched_rhs_shape));
+  auto expected_batched_rhs_shape = SymIntArrayRef(input.sym_sizes().data(), input.dim() - 1); // input.shape[:-1]
+  bool vector_case = other.dim() == 1 || (input.dim() - 1 == other.dim() && other.sym_sizes().equals(expected_batched_rhs_shape));
```

Comment:
I add the same function on the Python side below, but this one is needed for running `test_aot_autograd_symbolic_exhaustive_linalg_solve_ex_cpu_float32`.